example/example-etcd-cluster.yaml

```diff
@@ -5,3 +5,7 @@ metadata:
 spec:
   size: 3
   version: "3.1.0"
+  clusterTLS:
```

Comment:
Definitely, pushed a commit that scopes these fields. I did leave the `Name` suffix on the field names, to indicate it's a pointer to a secret and not a literal.